app/views/shelters/new.html.erb

```diff
@@ -0,0 +1,15 @@
+<form action="/shelters" method="post" class="new-shelter">
+    <input type="hidden" name="authenticity_token" value="<%=form_authenticity_token %>">
+    <p>Enter information for a new shelter:</p>
+    <label>Name</label>
+    <input type="text" name='shelter[name]' required /><br/>
+    <label>Address</label>
+    <input type="text" name='shelter[address]' required/><br/>
+    <label>City</label>
+    <input type="text" name='shelter[city]' required/><br/>
+    <label>State</label>
+    <input type="text" name='shelter[state]' required/><br/>
+    <label>Zip</label>
+    <input type="text" name='shelter[zip]' required/><br/>
```

Comment:
Inputs have been made required so they can't be submitted without filling in the necessary information.